bin/tenderly-debug

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ue
+
+cd docker
+
+file=./tenderly.yaml
+if [ ! -e "$file" ]; then
```

Comment:
i think you can just put ./tenderly.yaml directly in the if statement without the variable here